eras/alonzo/impl/cardano-ledger-alonzo.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-alonzo
-version:            1.6.0.1
+version:            1.6.1.1
```

You also need to change the version in the CHANGELOG as well
```suggestion
version:            1.6.1.0
```